src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const tableNames = document.querySelectorAll('thead th');
```

Comment:
Ensure that the `thead th` elements are correctly selected. If the table structure changes, this selector might need to be updated.